modules/cli/src/main/scala/coursier/cli/params/DependencyParams.scala

```diff
@@ -61,34 +61,17 @@ object DependencyParams {
             )
       }
 
-    val fromFilesDependenciesV: ValidatedNel[String, Seq[String]] =
-      if (options.dependencyFile.isEmpty)
-        Validated.validNel(Seq.empty)
-      else {
-        val files = options.dependencyFile
-        val fromFileDependencies = files.toSeq.validationNelTraverse { file =>
-          val validated = Try {
-            val source = Source.fromFile(file)
-            val lines =
-              try source.mkString.split("\n").filter(_.nonEmpty)
-              finally source.close()
-            lines.toSeq
-          }.toEither
-          ValidationNel.fromEither(validated)
-        }
-
-        fromFileDependencies.either match {
-          case Left(errors) =>
-            Validated.invalidNel(
-              s"Cannot read dependencies from files:" + System.lineSeparator() +
-                errors
-                  .map("  " + _.getMessage())
-                  .mkString(System.lineSeparator())
-            )
-          case Right(lines) =>
-            Validated.validNel(lines.flatten)
+    val fromFilesDependencies: Seq[String] =
```

Comment:
@alexarchambault why remove the validation? Wasn't it useful for some better errors? That was done in response to @julienrf 's suggestions